lib/minfraud/components/custom_inputs.rb

```diff
@@ -0,0 +1,14 @@
+module Minfraud
+  module Components
+    class CustomInputs < Base
+      # Creates Minfraud::Components::CustomInputs instance
+      # @param  [Hash] params hash with keys that match your created custom input keys
+      # @return [Minfraud::Components::CustomInputs] a CustomInputs instance
+      def initialize(params = {})
+        params.each do |param_name, param_value|
```

Comment:
How about renaming `param_name` and `param_value` to `name` and `value` respectively to avoid redundancy?
I believe that we can neglect the prefix due to the fact that we're iterating over `params` hash and it's kinda obvious.